zerver/lib/rest.py

```diff
@@ -152,6 +153,21 @@ def rest_dispatch(request: HttpRequest, /, **kwargs: object) -> HttpResponse:
     # Security implications of this portion of the code are minimal,
     # as we should worst-case fail closed if we miscategorize a request.
 
+    def check_is_authenticated(request: HttpRequest) -> bool:
+        if "narrow_user_cache" not in view_flags:
```

Comment:
Note sure this is the best name for this.

Reply:
I think these names are OK, resolving.